client/my-sites/plans-v2/style.scss

```diff
@@ -15,6 +15,19 @@
 		& > .details__column:not( :first-of-type ) {
 			margin-left: 24px;
 		}
+		.formatted-header {
+			margin-top: 40px;
+		}
+		.formatted-header__title {
+			font-size: 1rem;
+			color: #3c434a;
+			@media ( min-width: 660px ) {
```

Comment:
It seems like we're already in a media query at this point (`@include break-medium` on line 4).